components/rating.css

```diff
@@ -0,0 +1,35 @@
+.rating {
+  display: flex;
+  flex-direction: row-reverse;
+}
+
+.rating input,
+.static-rating input {
+  display: none;
+}
+
+.rating label,
+.static-rating label {
+  color: var(--bg-rating);
+  cursor: pointer;
+  font-size: 1.2rem;
+  margin: auto 0.5rem;
+}
+
+.static-rating label {
+  cursor: auto;
+}
+
+.rating label:hover,
+.rating label:hover ~ label {
+  color: #ffb3009c;
```

Comment:
Aren't HEX colors supposed to be of hexadecimal value (6 characters)
seems to be a typo
Also prefer to store the colorin a variable

Reply:
This color is used only once in hence not storage into the variable